.env.test

```diff
@@ -9,7 +9,7 @@ AWS_UNLINK_QUEUE_URL=
 AWS_HEARING_RESULTED_QUEUE_URL=
 AWS_PROSECUTION_CONCLUDED_QUEUE_URL=
 
-INLINE_SIDEKIQ=true
+INLINE_SIDEKIQ=false
```

Comment:
There was a global `Sidekiq::Testing.fake!` hidden in one of the now-deleted spec files that caused sidekiq to _not_ be inline for the entire test suite. To avoid breaking all the tests, I've updated this to have the same effect as the now-deleted config call.